src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -176,34 +176,44 @@ private GATKRead applyREVERT_SOFTCLIPPED_BASES(final GATKRead read) {
     }
 
     /**
-     * Given a cigar string, get the number of bases hard or soft clipped at the start
+     * Given two cigar strings corresponding to the a read that has been softcliped and before, returns an integer
+     * corresponding to the number of reference bases that the new string must be offset by in order to have the
+     * correct start according to the reference.
+     *
+     * @VisibleForTesting
```

Comment:
`@VisibleForTesting` should be outside of the comment, and before the method header